extra/1-currency-conversion.js

```diff
@@ -15,7 +18,11 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(pound) {
+  const fee = pound / 100;
+  const exchange = (pound - fee) * 5.7;
+  return +exchange.toFixed(2);
```

Comment:
Is there a reason why you have used a unary plus sign here?

Reply:
it's return string so I use + sign instead Number() to change string to number